app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -324,7 +326,7 @@ def parse_persistent_volume(persistent_volume)
       new_result.merge!(
         :type           => 'PersistentVolume',
         :parent_type    => 'ManageIQ::Providers::ContainerManager',
-        :capacity       => persistent_volume.spec.capacity.to_h.map { |k, v| "#{k}=#{v}" }.join(','),
+        :capacity       => parse_capacity(persistent_volume.spec.capacity.to_h),
```

Comment:
@alongoldboim we should use the same names as kubernetes for as much as possible here. I think this should be called `parse_resource_list`.

You don't have to do that in this PR (you can rename this later), but let's not forget.